### ISSUE TITLE
GitLink doesn't have any PDBs to work with because of a packaging defect 

### ISSUE THREAD

**Issue by PureWeen** [reactiveui/ReactiveUI#1254]
Looking at the packages that get created it appears that the PDBs don't get packaged up so GitLink isn't able to apply it's magic.

Some notes from Slack

"GitLink is setup; if it's not working then that's an defect.

[3:32]  
(fyi you're first to mention GitLink since introduced in V7.0.0 so yeah good chance something might be broken)

[3:33]  
Ya looks like a packaging defect.

[3:35]  
Can you open up an GitHub issue @pureween. Maybe even PR the nuspecs. Right now only DLL is explicitly defined. Can you duplicate each line and add reference to pdb? Don't use wildcards as they fail silently.

[3:37]  
Hmm. Double check that's the right course of action as well. It's late. :)"

**Comment by PureWeen** [comment]
From other nuspecs I saw that appears to be how you do it . I didn't find anything magical that would just include those in their alongside the DLLs.  

I did a quick test locally with those added on the rxui-core nuspec and the srcsrv gets put into the pdb file so it seems like it'll all wire up nicely.

I'll do a PR on the nuspecs here in the next couple days. Is there any interest in also including the XML files? Figured while I was in there I'd ask.
